README.md

```diff
@@ -1,5 +1,57 @@
+
+Everything chosen for the next year’s project must be defined in a sufficiently mature standards-track web specification. If there is no web standard, then there is no definition of what a particular technology “should” be doing. The Interop Project is not an organization for defining web standards. That work is important, and must happen in the appropriate bodies, not here.
+
+Everything chosen for the next year’s project must be testable using automated tests on existing testing platforms. A proposal cannot be accepted if there are no automated tests to evaluate conformance to the web standard; if it’s not possible to test the technology being proposed; or if the tests would need to run in environments that are not yet available on the infrastructure for Web Platform Tests (for example, in 2022 and 2023, testing on mobile operating systems was not possible).
+
+If there is a particular blocker making it hard to test a certain feature, that could make for a good Investigation Project proposal. And ideas about how the WPT/Interop Project testing infrastructure can be improved for the future can be proposed as Investigation Projects, to be considered as work for the Interop Project to take on in the next year.
+
+### **Requirements for Focus Area proposals**
+
+* This feature has a mature-enough web standard from an established standards development organization such as IETF, Khronos Group, TC39, WHATWG, or the W3C.
+* This feature has high quality tests in WPT or Test262.
+
+### **Guidance for Prioritizing Focus Area proposals**
+
+Beyond the above requirements, in line with the above goal, areas where web developers have had problems using them should be prioritized, whether that is because of differences between browsers, or because they aren’t implemented in all browsers. There are many potential sources for information about such areas, including but not limited to:
+
+* Issues filed about such problems in the Chromium, Gecko, or WebKit bug trackers.
+* Complaints about this feature in public discussions on sites like Stack Overflow.
+* Ranking as a high-demand feature or as a pain-point consistently in polls and surveys of web developers.
+* Documented examples of sites or libraries working around the fact that a feature is missing or not interoperable.
+
+When assessing the prioritization of existing web technology, it can be helpful to assess how often the technology is being used currently. However, one should also consider if important technology might not be in popular use due to lack of interoperability. Potential sources of data include but are not limited to:
+
+* Browser use counters.
+* Usage recorded via HTTP Archive crawls.
+* Number of mentions on developer resources like Stack Overflow.
+
+There are also guiding values that should be taken into consideration, beyond pure demand:
```

Comment:
I think we should remove ", beyond pure demand" because it makes it sound like these priorities are less important than demand.